tests/sentry/dynamic_sampling/test_tasks.py

```diff
@@ -0,0 +1,18 @@
+from collections import defaultdict
+
+import pytest
+
+from sentry.dynamic_sampling.tasks import fetch_projects_with_total_volumes
+
+
+@pytest.mark.django_db
+def test_simple_no_data(default_project):
```

Comment:
Are you planning on adding more tests for the tasks?

Reply:
done.